test/gurka/gurka.h

```diff
@@ -538,18 +547,6 @@ findEdge(const std::unique_ptr<valhalla::baldr::GraphReader>& reader,
   return std::make_tuple(baldr::GraphId{}, nullptr, baldr::GraphId{}, nullptr);
 }
 
-std::tuple<const baldr::GraphId,
-           const baldr::DirectedEdge*,
-           const baldr::GraphId,
-           const baldr::DirectedEdge*>
-findEdge(const map& map,
-         const baldr::GraphId& tile_id,
-         const std::string& way_name,
-         const std::string& end_node) {
-  std::unique_ptr<baldr::GraphReader> reader(new baldr::GraphReader(map.config));
```

Comment:
this doesnt make sense. if you instantiate a reader and then return a pointer to memory that it holds but then it goes out of scope and is deallocated, the pointer is no longer valid. instead of this i just removed unique_ptr from both the findEdges functions and pass a reference to a graphreader so the memory will be gauranteed to still be there